Vic2ToHoI4/Source/HOI4World/HoI4Country.cpp

```diff
@@ -112,24 +121,24 @@ HoI4::Country::Country(std::string tag,
 
 
 HoI4::Country::Country(const std::string& tag_,
-	 const Country& owner,
+	 const std::shared_ptr<Country> owner,
 	 const std::string& region_,
 	 const Regions& regions,
 	 Mappers::GraphicsMapper& graphicsMapper,
 	 Names& names,
 	 Localisation& hoi4Localisations):
 	 tag(tag_),
-	 primaryCulture(owner.primaryCulture), primaryCultureGroup(owner.primaryCultureGroup), civilized(owner.civilized),
-	 rulingParty(owner.rulingParty), parties(owner.parties), upperHouseComposition(owner.upperHouseComposition),
-	 lastElection(owner.lastElection), color(owner.color), graphicalCulture(owner.graphicalCulture),
-	 graphicalCulture2d(owner.graphicalCulture2d), warSupport(owner.warSupport),
-	 oldTechnologiesAndInventions(owner.oldTechnologiesAndInventions), atWar(owner.atWar), shipNames(owner.shipNames),
-	 generatedDominion(true), region(region_), puppetMaster(owner.getTag()), puppetMasterOldTag(owner.getOldTag()),
-	 governmentIdeology(owner.getGovernmentIdeology()), leaderIdeology(owner.getLeaderIdeology()), oldCapital(-1)
+	 primaryCulture(owner->primaryCulture), primaryCultureGroup(owner->primaryCultureGroup), civilized(owner->civilized),
+	 rulingParty(owner->rulingParty), parties(owner->parties), upperHouseComposition(owner->upperHouseComposition),
+	 lastElection(owner->lastElection), color(owner->color), graphicalCulture(owner->graphicalCulture),
+	 graphicalCulture2d(owner->graphicalCulture2d), warSupport(owner->warSupport),
+	 oldTechnologiesAndInventions(owner->oldTechnologiesAndInventions), atWar(owner->atWar), shipNames(owner->shipNames),
+	 generatedDominion(true), region(region_), puppetMaster(owner), puppetMasterOldTag(owner->getTag()),
```

Comment:
This should be ```getOldTag()``` though really could even be removed now that we're storing the whole puppet master.